modules/programs/firefox.nix

```diff
@@ -86,6 +86,17 @@ in
           then pkgs.firefox
           else pkgs.firefox-unwrapped;
         defaultText = literalExample "pkgs.firefox";
+        example = literalExample ''
+          pkgs.firefox.override {
+            # See nixpkgs' firefox/wrapper.nix to check which options you can use
```

Comment:
Is it worth making this an actual link to https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/browsers/firefox/wrapper.nix#L56?

Reply:
Sure, do you want it as a comment instead of what I initially wrote?

Reply:
now that I look at it again, since this is inside a literalExample, I don't think it makes much sense to have a full-out link. Let's skip for now, unless others have ideas for how to handle this.

Reply:
The thing is we wouldn't want to have to update this again if the option disappears.

Reply:
That's what the comment is for, to me. To explain that it is only an illustrative example.